### PR TITLE
Add Organization Management Documentation

### PR DESCRIPTION
**Pull request by vipulgupta2048**
While writing this piece, I tried having a sanity check in place for each section while I was writing. This helps the user not wander all over the UI or experiment with buttons rather exactly know the expected outcome behind an action. Let me know if at places I have given too much or too little information for a particular step.

**Sanity Check**
- What is the feature?
- How can I do that (instructions or steps)
- Who can do it? (Admins or members)
- What will happen next?
- Screenshot to guide users

**Todo**

- [x] Add screenshots to the blog - The UI is still updating, so I thought it be best to take screenshots at the very end. 
- [ ] Organizations is a significant change that we need to brace for, a lot of screenshots around balenaCloud dashboard might need to be updated with the new `create application` modal, or change to how breadcrumbs look and many more. These will be our next chore.
- [ ] Update screenshot --> https://www.balena.io/docs/learn/manage/app-types/
- [x] Fix links and generate documentation to check if they are working
- [x] Update behaviors and add more questions at the end of this page after everyone's review 

Change-type: patch
Signed-off-by: Vipul Gupta (@vipulgupta2048) <vipul@balena.io>

### CODE REVIEW

File: pages/learn/manage/organizations.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+
+## Creating a new organization
+
+To create a new organization, navigate to your balenaCloud dashboard and click the `Organizations` tab on the sidebar.
+
+<img alt="Organizations tab" src="/img/common/app/organizations_tab.png" width="100%">
+
+Next, click the `Create Organization` button to get the create organization modal below. Fill in the name of your organization and you are all set. This is also the page where you can check all organizations that you have access to. The user creating an organization is default given administrator access to that organization. Only administrators can delete an organization. Alternatively, anyone can leave an organization by clicking the `Leave organization` button on the organization summary page.
+
```
Comment (pdcastro) [inline pages/learn/manage/organizations.md:21]: `is default` → `is by default`

File: pages/learn/manage/organizations.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+### Inviting members to an organization
+
+Now, with your organization created let's invite some members. Only the administrator can invite users to an organization. To invite, click the `Manage members` button on the organization summary dashboard. On the page that opens, click the `Invite member` button to open the `Invite organization member` modal. Here, you can either add users with their email or with their balena username. After assigning them roles in your organization. Click `Add member` to invite the user to your organization.
+
+<img alt="Invite Organization Member Modal" src="/img/common/app/" width="100%">
+
+On basis of the role assigned, they will be able to access, see, and manage both apps and other members inside the organization. To change the roles of a member, navigate to the `Organization Members` tab in the sidebar and change the `Organization role` of the desired user by using the dropdown list against their username.
```
Comment (pdcastro) [inline pages/learn/manage/organizations.md:40]: `On basis of` → `On the basis of`
_or_ `Depending on the role assigned,`

File: pages/learn/manage/organizations.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+<img alt="Change Organization Member Roles" src="/img/common/app/" width="100%">
+
+
+## Creating teams in your organization
+
+With organizations, you can group your existing members to form teams as well. Teams are a robust way to manage members per their function and bulk provide access to applications as needed. Only administrators are allowed to create a team. To form a team, navigate to `Organization Teams` in the sidebar of the organization summary page. On the page that opens, click the `Create Team` button and enter the name of the team that is being created.
```
Comment (pdcastro) [inline pages/learn/manage/organizations.md:47]: * `per` → `by`
* `and bulk provide access` → `and to provide access in bulk`

File: pages/learn/manage/organizations.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+## Creating new applications under your organization
+
+With members invited to the organization, let's get started on creating new applications under an organization. Navigate to the `Organization Applications` tab in the sidebar to find the `Create application` button for creating a new application under your organization. Learn more about creating an application from the [getting started](getting-started) guide.
+
+<img alt="New Application under Organization" src="/img/common/app/" width="100%">
+
+### Moving existing applications to an organization
+
+Moving an existing application from your personal or any other organization to another organization is a common task that can be accomplished by [transferring ownership](transferring-ownership) of your application.
```
Comment (pdcastro) [inline pages/learn/manage/organizations.md:64]: `from your personal or any other organization to another organization`
→ `to an organization`
I.e.,
`Moving an existing application to an organization is a common task ...`

File: pages/learn/manage/organizations.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+<img alt="Transfer Ownership" src="/img/common/app/" width="100%">
+
+### Providing access to multiple applications
+
+With organizations, providing access to individual members or teams becomes a hassle-free process. There are mainly 3 application-level roles that can be assigned, [`Developer`](developer), [`Observer`](observer) or [`Operator`](operator). Administrator default to [`Owner`](owner) role to all applications under a organization and are the only ones that can provide access to applications. Administrators won't be listed in members page of an organization.
```
Comment (pdcastro) [inline pages/learn/manage/organizations.md:70]: * `or Operator` → `and Operator`
* `Administrator default to [`Owner`](owner) role to all applications`
  → `Administrators have the [`Owner`](owner) role by default for all applications`

File: pages/learn/manage/organizations.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+
+<img alt="Transfer Ownership" src="/img/common/app/" width="100%">
+
+### Providing access to multiple applications
+
+With organizations, providing access to individual members or teams becomes a hassle-free process. There are mainly 3 application-level roles that can be assigned, [`Developer`](developer), [`Observer`](observer) or [`Operator`](operator). Administrator default to [`Owner`](owner) role to all applications under a organization and are the only ones that can provide access to applications. Administrators won't be listed in members page of an organization.  
+
+- **Providing access to members** - Once you are done inviting members, click the username you want to provide access to from the `Members` list. When the member page opens, click `Manage applications` to see all applications available under the org. From there, click the `Add applications to member` button to add the member to one or more applications available under the org with the desired role that you want to assign.
```
Comment (pdcastro) [inline pages/learn/manage/organizations.md:72]: * `member page` → `member's page`
* delete `that you want to assign`

File: pages/learn/manage/organizations.md [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+
+<img alt="Provide Member Access to Applications" src="/img/common/app/" width="100%">
+
+- **Providing access to teams** - Once you are done creating a team, navigate to the team you want to provide access in the `Organization Teams` tab.  Next, click the `Manage applications` button on the team page to see all applications available under the org. From there, click the `Add applications to team` button to add the team to one or more applications available under the org with the desired roles that you want to assign. All members of that team will be provided access to the selected application with the specified role per application.
```
Comment (pdcastro) [inline pages/learn/manage/organizations.md:76]: delete `that you want to assign`

File: pages/learn/manage/organizations.md [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+
+<img alt="Provide Team Access to Applications" src="/img/common/app/" width="100%">
+
+
+## Users as an organization - Sharing your apps just became hassle-free
```
Comment (pdcastro) [inline pages/learn/manage/organizations.md:81]: `## Users as an organization - hassle-free app sharing`

File: pages/learn/manage/organizations.md [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+
+With organizations being a crucial feature for collaboration, we wanted the same for individual users. Each user in balenaCloud has their own "personal" organization named after their username. This "personal organization" with all applications and devices under it are private until explicitly shared by the user.
+
+Personal organizations can be used to develop & test applications. When ready, these applications can be shared with other members or teams by inviting users to your organization. A personal organization is no different from any other organization that you can create and works in the same way. With this, each user is empowered to collaborate, manage and work on their fleets more easily & securely.
```
Comment (pdcastro) [inline pages/learn/manage/organizations.md:85]: `With this,` → `This way,`

File: pages/learn/manage/organizations.md [hunk 10]
```diff
@@ -0,0 +1,97 @@
+---
+title: Organization management
+---
+
+# Organization management
+
+Organizations in balenaCloud are based around the idea of easier collaboration and robust management of members and applications. The feature blends in with current workflows all the while improving what is already there.
```
[the remark this answers is not in the document]
Comment (shaunmulligan) [reply pages/learn/manage/organizations.md:7]: Agree with Paulo on above. We want a bit more info on what they are and what they will be used for up front

File: pages/learn/manage/organizations.md [hunk 11]
```diff
@@ -0,0 +1,97 @@
...
+
+## Sign up
```
Comment (shaunmulligan) [inline pages/learn/manage/organizations.md:9]: Maybe this feels a bit too obvious? I would probably just remove it.

File: pages/learn/manage/organizations.md [hunk 12]
```diff
@@ -0,0 +1,97 @@
...
+<img alt="Provide Member Access to Applications" src="/img/common/app/" width="100%">
+
+- **Providing access to teams** - Once you are done creating a team, navigate to the team you want to provide access in the `Organization Teams` tab.  Next, click the `Manage applications` button on the team page to see all applications available under the org. From there, click the `Add applications to team` button to add the team to one or more applications available under the org with the desired roles that you want to assign. All members of that team will be provided access to the selected application with the specified role per application.
+
+<img alt="Provide Team Access to Applications" src="/img/common/app/" width="100%">
+
+
+## Users as an organization - Sharing your apps just became hassle-free
+
+With organizations being a crucial feature for collaboration, we wanted the same for individual users. Each user in balenaCloud has their own "personal" organization named after their username. This "personal organization" with all applications and devices under it are private until explicitly shared by the user.
```
Comment (shaunmulligan) [inline pages/learn/manage/organizations.md:83]: I believe we don't want to call it "personal" but rather "default" org.

**Review by shaunmulligan — COMMENTED** [review]
@vipulgupta2048 and @jtonello only had a few comments and echoes of what Paulo said, but other wise looks good.

File: pages/learn/manage/organizations.md (hunk 11 above)
Comment (vipulgupta2048) [reply pages/learn/manage/organizations.md:9]: Roger that, added it because I was following the same template as account management document

File: pages/learn/manage/organizations.md (hunk 12 above)
Comment (vipulgupta2048) [reply pages/learn/manage/organizations.md:83]: Yes, that's a discussion-worthy thing we need to take a decision on the blog as well, I will check in on that before making a decision.

Comment (vipulgupta2048) [reply pages/learn/manage/organizations.md:78]: @shaunmulligan We will be going with "default" org everywhere.

File: pages/learn/manage/organizations.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (vipulgupta2048) [reply pages/learn/manage/organizations.md:7]: I agree @pdcastro the introduction paragraph was something I was holding off for later in the view of plucking a good definition out of the blog John is writing. But, yes this is needed and extremely important for folks to read when they land on the page. I will keep this open, and post what we are adding for a definition.

[the remark this answers is not in the document]
Comment (jtonello) [reply pages/learn/manage/organizations.md:7]: I like the core of what Paulo wrote, but we should add more of the "Why" to the description. For example, "orgs make it easier for teams of developers to collaborate across one or many applications", "orgs allow you to create better distinctions between development and production projects)", "orgs allow more specific access and views of project and codes to help meet security and regulatory rules", etc.

[the remark this answers is not in the document]
Comment (vipulgupta2048) [reply pages/learn/manage/organizations.md:7]: @jtonello  You got it, should we also link back to the blog post. Since that covers a lot of the why and other questions that might pop in people's head when going through the feature.

[the remark this answers is not in the document]
Comment (vipulgupta2048) [reply pages/learn/manage/organizations.md:7]: Screenshot from 2020-12-04 13-13-35

How's this looking @pdcastro @shaunmulligan @jtonello Will push this as well.

File: pages/learn/manage/organizations.md [hunk 13]
```diff
@@ -0,0 +1,100 @@
+---
+title: Organization management
+---
+
+# Organization management
+
+An **organization** in balenaCloud is a set of users (members and roles), teams, and applications. Which in turn make collaboration easier and access control more streamlined than ever for your applications and devices. Organizations are available for use in all balenaCloud accounts, blending in with existing features and workflows all the while empowering fleet owners to build bigger, better managed fleets.
```
[the remark this answers is not in the document]
Comment (vipulgupta2048) [reply pages/learn/manage/organizations.md:7]: @pdcastro We haven't released organizations yet to all users. That's the reason you aren't able to see it at this point from your free account (Whether staging or prod). Alison did mention when organizations will be released the feature will be available to all users and won't be part of paid plans. So that's that. 

I do agree documentation needs to be more low key. That's the reason I added the `What can you do` section below the introduction paragraph where the points are clear with no keywords, no-frills. But, yes I agree I have gone too far with that one. :joy:

[the remark this answers is not in the document]
Comment (pdcastro) [reply pages/learn/manage/organizations.md:7]: Oh, I missed the new `What can you do` section, sorry. There is some overlap between my suggestion above and the new section, so it needs to be changed a bit.

File: pages/learn/manage/organizations.md [hunk 14]
```diff
@@ -0,0 +1,100 @@
...
+
+## What can you do with organizations?
+
+With organizations you can:
+
+* Invite users to an orgnization as members.
+* Create teams of those members for easier collaboration across applications.
+* Bulk provide explicit access to applications to individual members or teams now. 
+* Manage each member, team, or application with easier controls to help meet security and regulatory rules.
```
Comment (pdcastro) [inline pages/learn/manage/organizations.md:16]: _"Manage X with easier controls"_ is not exactly what helps meet security and regulatory rules... What helps meet security and regulatory rules is the ability set access control to apps at a granular level, with user roles like observer / operator / developer, and the ability to change and revoke such roles. I think...

File: pages/learn/manage/organizations.md [hunk 15]
```diff
@@ -0,0 +1,100 @@
...
+# Organization management
+
+An **organization** in balenaCloud is a set of users (members and roles), teams, and applications. Which in turn make collaboration easier and access control more streamlined than ever for your applications and devices. Organizations are available for use in all balenaCloud accounts, blending in with existing features and workflows all the while empowering fleet owners to build bigger, better managed fleets.
+
+## What can you do with organizations?
+
+With organizations you can:
+
+* Invite users to an orgnization as members.
+* Create teams of those members for easier collaboration across applications.
```
Comment (pdcastro) [inline pages/learn/manage/organizations.md:14]:
```
* Invite users to an organization as members, and group them in teams.
* Manage access control at the team level, associating applications with teams.
```

File: pages/learn/manage/organizations.md (hunk 13 above)
[the remark this answers is not in the document]
Comment (vipulgupta2048) [reply pages/learn/manage/organizations.md]: @pdcastro How's it looking now? With screenshots too.

File: pages/learn/manage/organizations.md [hunk 16]
```diff
@@ -0,0 +1,103 @@
+---
+title: Organization management
+---
+
+# Organization management
+
+An **organization** in balenaCloud is a group of members, teams, and applications with control over accessiblity and visiblity. Organizations allow granular access control at the application level through multiple roles that can be assigned to individual nembers or teams. This in turns helps to improve the visibility of who has access to which applications making collaboration easier.
```
Comment (pdcastro) [inline pages/learn/manage/organizations.md:7]: * The words "accessiblity", "visiblity", "nembers" have typos in them - it would be a good idea to run a spell checker over the whole document.
* I'd rephrase _"with control over accessibility and visibility"_ (especially in the first sentence of this document) because "accessibility" generally refers to making computer systems accessible to people with disabilities or impairments to vision, hearing, etc, such as using larger fonts, high contrast, closed captions in video, etc. I think you meant access control instead of accessibility. _"control over visibility"_ leaves one wondering, _"visibility over what?"_ or _"why would I want to control visibility?"..._
* _"a group of members" (especially in the first sentence of this document) may sound puzzling to readers -- members of what, what members? -- not realizing that member means a balenaCloud user.

Suggestion:
```
An **organization** in balenaCloud is a group of users (members), teams, and applications,
with an access control system. Organizations allow the assignment of access control roles
to members or teams, improving visibility and reasoning over who has access to what applications
and, consequently, to device fleets.
```

By the way, in our quest for the soul of organizations, is an organization an access control system? Where "resources" are applications and "actors" are users and teams (which are groups or users). In which case:

```
An _organization_ in balenaCloud is an access control system for balena applications.
Organization _members_ are balenaCloud users who may be grouped in _teams,_ and
assigned _roles_ such as developer or operator. Organizations improve visibility and
reasoning over who has access to what applications and, consequently, to device fleets.

A balenaCloud account may contain one or more organizations, but a balena application
may be added to a single organization.
```

File: pages/learn/manage/organizations.md [hunk 17]
```diff
@@ -0,0 +1,103 @@
+---
+title: Organization management
+---
+
+# Organization management
+
+An **organization** in balenaCloud is a group of members, teams, and applications with control over accessiblity and visiblity. Organizations allow granular access control at the application level through multiple roles that can be assigned to individual nembers or teams. This in turns helps to improve the visibility of who has access to which applications making collaboration easier. 
+
+Organizations are available for use in all balenaCloud accounts, and multiple organizations can be created by a single user. Organizations blend in with existing features and workflows improving upon what's already there.
```
Comment (pdcastro) [inline pages/learn/manage/organizations.md:9]: A sentence like _"Organizations blend in with existing features and workflows improving upon what's already there"_ is not really helpful to readers of this document in my opinion, and could be deleted.

_"multiple organizations can be created by a single user"_ does not make it clear whether such a user is using one or more balenaCloud user accounts. I think the following is a more useful fact to be documented (as always, assuming it is correct...).

```
A balenaCloud account may contain one or more organizations, but a balena application
may be added to a single organization.
```

File: pages/learn/manage/organizations.md (hunk 16 above)
Comment (vipulgupta2048) [reply pages/learn/manage/organizations.md:7]: My bad on the typos, I have checked the document before but will run the checker just to be safe. I liked both your suggestions so how about this as the final version. Pushed it just now.

**Comment by pdcastro** [comment]
@vipulgupta2048, I'm happy to approve the PR. 👍   If I do, it gets merged straight away, hence this question: Are you planning to make any final changes, such as squashing commits?

**Comment by vipulgupta2048** [comment]
@pdcastro I would say let's wait for Monday. I would like to get some more eyes + we can release this with the team together. Otherwise looks pretty good. I will do some commit clean up and go through them again. Some button texts in the UI has changed when I wrote the first iteration of these docs and I want to be sure before merging. Thanks for the help in reviewing this!

File: pages/learn/manage/organizations.md [hunk 18]
```diff
@@ -0,0 +1,104 @@
...
+
+<img alt="Provide Team Access to Applications" src="/img/common/app/application_access_to_team.png" width="100%">
+
+
+## Users as an organization - hassle-free app sharing
+
+With organizations being a crucial feature for collaboration, we wanted the same for each individual user too. Hence, every user in balenaCloud owns an organization by default named after their username. In this **default organization**, all applications and devices are private until **explicitly shared** by the user.
+
+Default organizations can be used to develop & test applications. When ready, these applications can be shared with other members or teams by inviting users to your organization. A default organization is no different from any other organization that you can create. This way, each user is empowered to collaborate, manage, and work on their fleets securely.
```
Comment (dfunckt) [inline pages/learn/manage/organizations.md:92]: 😁

```suggestion
```

File: pages/learn/manage/organizations.md [hunk 19]
```diff
@@ -0,0 +1,104 @@
+---
+title: Organization management
+---
+
+# Organization management
+
+An **organization** in balenaCloud is a group of users (members), teams, and applications,
+with an access control system. Organization _members_ are balenaCloud users who may be grouped into _teams_, and can be assigned _roles_ such as developer, observer, or operator in balenaCloud applications. Organizations essentially improve visibility and reasoning over who has access to which applications and, consequently, to which device fleets.
+
+The feature is available for use in all balenaCloud accounts irrespective of plans. A balenaCloud account may contain one or more organizations.
```
Comment (dfunckt) [inline pages/learn/manage/organizations.md:10]:
```suggestion
In balenaCloud, users organize around and collaborate in _organizations_. An organization is a group of users, teams and applications, and is managed by one or more organization administrators. An organization can have any number of members that can be grouped into teams and granted varying levels of access to its applications according to their assigned role.

Administrators are implicitly granted the ability to perform any possible action to the organization and its associated resources. This includes inviting and removing members, managing applications, teams and billing.

Each user can create or be a member of an unlimited number of organizations. An organization is automatically created upon user registration, and that user is assigned as its administrator. This organization is managed like any other, but currently cannot be deleted.

Billing applies to each organization separately. Administrators can at any time upgrade an organization to the appropriate billing plan as needed.
```

Sorry, I started trying to improve this blurb a bit but ended up with a bunch of text. Feel free to edit and split it up and distribute in the appropriate sections as you like.

I tried to refine the "organization" definition and highlight admins and billing.

File: pages/learn/manage/organizations.md [hunk 20]
```diff
@@ -0,0 +1,104 @@
...
+<img alt="Invite Organization Member Modal" src="/img/common/app/invite_org_member.png">
+
+Depending on the role assigned, they will be able to access, see, and manage both apps and other members inside the organization. To change a member's role, navigate to the `Organization Members` tab in the sidebar. Next, change the `Organization role` of the desired user using the dropdown list against their username.
+
+<img alt="Change Organization Member Roles" src="/img/common/app/modify_org_roles.png" width="100%">
+
+
+## Creating teams in your organization
+
+With organizations, you can group your existing members to form teams as well. Teams are a robust way to manage members by their function, and bulk provides access to applications as needed. Only administrators are allowed to create a team. To form a team, navigate to the `Organization Teams` tab in the sidebar. On the page that opens, click the `Create Team` button and enter the team's name that is being created.
```
Comment (dfunckt) [inline pages/learn/manage/organizations.md:54]:
```suggestion
Organization members can be grouped into teams. Teams provide a way to manage members according to their function. Only administrators can create and delete teams, and manage their members.

To create a team, navigate to the `Organization Teams` page via the sidebar and click the `Create Team` button to enter a name for the team.
```

File: pages/learn/manage/organizations.md [hunk 21]
```diff
@@ -0,0 +1,104 @@
...
+
+<img alt="Create Team in Organization" src="/img/common/app/create_team.png" width="100%">
+
+When the team is created, you can start adding members and applications to that specific team through the `Manage members` and `Manage applications` buttons, respectively.
```
Comment (dfunckt) [inline pages/learn/manage/organizations.md:58]:
```suggestion
Once the team is created, add members and grant the team access to applications by clicking on the `Manage members` and `Manage applications` buttons, respectively.
```

File: pages/learn/manage/organizations.md [hunk 22]
```diff
@@ -0,0 +1,104 @@
...
+
+<img alt="Organization Team Summary" src="/img/common/app/org_team_summary.png" width="100%">
+
+
+## Creating new applications under your organization
+
+With members invited to the organization, let's get started on creating new applications under an organization. Navigate to the `Organization Applications` tab in the sidebar to find the `Create application` button for creating a new application under your organization. Learn more about creating an application from the [getting started](getting-started) guide.
```
Comment (dfunckt) [inline pages/learn/manage/organizations.md:65]: I may be misreading the document, but it feels like it can't decide whether its function is to provide reference-level documentation or be a kind of "getting started with orgs" guide. It's perhaps okay as a start, but I personally think we should revisit soon after release.

File: pages/learn/manage/organizations.md [hunk 23]
```diff
@@ -0,0 +1,104 @@
...
+
+<img alt="New Application under Organization" src="/img/common/app/create_app_under_org.png" width="100%">
+
+### Moving existing applications to an organization
+
+Moving an existing application to an organization is a common task accomplished by [transferring ownership](transferring-ownership) of your application.
+
+<img alt="Transfer Ownership" src="/img/common/app/transfer_ownership.png" width="100%">
+
+### Providing access to multiple applications
```
Comment (dfunckt) [inline pages/learn/manage/organizations.md:75]:
```suggestion
### Granting members access to applications
```

File: pages/learn/manage/organizations.md [hunk 24]
```diff
@@ -0,0 +1,104 @@
...
+
+With organizations, providing access to individual members or teams becomes a hassle-free process. There are mainly three application-level roles that can be assigned, [`Developer`](developer), [`Observer`](observer), and [`Operator`](operator). Administrators have the [Owner](owner) role by default for all applications under an organization and are the only ones that can provide access to applications.
```
Comment (dfunckt) [inline pages/learn/manage/organizations.md:77]:
```suggestion
Organization administrators can grant access to applications either directly for individual members or for teams. There are three application-level roles that can be assigned, [`Developer`](developer), [`Observer`](observer), and [`Operator`](operator), which grant varying levels of access. Administrators are implicitly granted full access to all applications of the organization.
```

File: pages/learn/manage/organizations.md [hunk 25]
```diff
@@ -0,0 +1,104 @@
...
+
+- **Providing access to members** - Once you are done inviting members, click the username of the member you want to provide access to from the `Members` list. When the member's summary page opens, click `Manage applications` to see all applications available under the org. From there, click the `Add to applications` button to add the member to one or more applications available under the org with the desired role.
```
Comment (dfunckt) [inline pages/learn/manage/organizations.md:79]: Given the above blurb:

```suggestion
- **Granting access to applications for individual members** - Once you are done inviting members, click the username of the member you want to provide access to from the `Members` list. When the member's summary page opens, click `Manage applications` to see all applications available under the org. From there, click the `Add to applications` button to add the member to one or more applications available under the org with the desired role.
```

File: pages/learn/manage/organizations.md [hunk 26]
```diff
@@ -0,0 +1,104 @@
...
+
+<img alt="Provide Member Access to Applications" src="/img/common/app/application_access_to_member.png" width="100%">
+
+- **Providing access to teams** - Once you are done forming a team, navigate to the team, you want to provide access to in the `Organization Teams` tab.  Next, click the `Manage applications` button on the team page to see all applications available under the org. From there, click the `Add applications to team` button to add the team to one or more applications available under the org with the desired roles. All members of that team will be provided access to the selected application(s) with the specified role per application.
```
Comment (dfunckt) [inline pages/learn/manage/organizations.md:83]:
```suggestion
- **Granting access to applications for teams** - Teams and their members can be granted access to specific applications from the `Organization Teams` tab.  Click the `Manage applications` button on the team page to see all applications available under the org. From there, click the `Add applications to team` button to add the team to one or more applications available under the org with the desired roles. All members of that team will be provided access to the selected application(s) with the specified role per application.
```

File: pages/learn/manage/organizations.md (hunk 19 above)
Comment (vipulgupta2048) [reply pages/learn/manage/organizations.md:10]: No worries, @dfunckt I will add all these suggestions in the batch and see them as a complete document. If I have any other changes will mention here.

File: pages/learn/manage/organizations.md (hunk 22 above)
Comment (vipulgupta2048) [reply pages/learn/manage/organizations.md:70]: @dfunckt I started out with a purely referential piece of documentation complementing the blog we are writing but I agree in the end it seems to have a more a leaner, getting started guide tone.

File: pages/learn/manage/organizations.md [hunk 27]
```diff
@@ -0,0 +1,104 @@
...
+
+Moving an existing application to an organization is a common task accomplished by [transferring ownership](transferring-ownership) of your application.
+
+<img alt="Transfer Ownership" src="/img/common/app/transfer_ownership.png" width="100%">
+
+### Granting members access to applications
+
+Organization administrators can grant access to applications either directly for individual members or for teams. There are three application-level roles that can be assigned, [`Developer`](developer), [`Observer`](observer), and [`Operator`](operator), which grant varying levels of access. Administrators are implicitly granted full access to all applications of the organization.
+
+- **Granting access to applications for individual members** - Once you are done inviting members, click the username of the member you want to provide access to from the `Members` list. When the member's summary page opens, click `Manage applications` to see all applications available under the org. From there, click the `Add to applications` button to add the member to one or more applications available under the org with the desired role.
```
Comment (dfunckt) [inline pages/learn/manage/organizations.md:84]: Sorry, I think my previous suggestion is not proper English and maybe the following is better:

```suggestion
- **Granting individual members access to applications** - Once you are done inviting members, click the username of the member you want to provide access to from the `Members` list. When the member's summary page opens, click `Manage applications` to see all applications available under the org. From there, click the `Add to applications` button to add the member to one or more applications available under the org with the desired role.
```

File: pages/learn/manage/organizations.md [hunk 28]
```diff
@@ -0,0 +1,104 @@
...
+
+<img alt="Provide Member Access to Applications" src="/img/common/app/application_access_to_member.png" width="100%">
+
+- **Granting access to applications for teams** - Teams and their members can be granted access to specific applications from the `Organization Teams` tab.  Click the `Manage applications` button on the team page to see all applications available under the org. From there, click the `Add applications to team` button to add the team to one or more applications available under the org with the desired roles. All members of that team will be provided access to the selected application(s) with the specified role per application.
```
Comment (dfunckt) [inline pages/learn/manage/organizations.md:88]: Same here:

```suggestion
- **Granting teams access to applications** - Teams and their members can be granted access to specific applications from the `Organization Teams` tab.  Click the `Manage applications` button on the team page to see all applications available under the org. From there, click the `Add applications to team` button to add the team to one or more applications available under the org with the desired roles. All members of that team will be provided access to the selected application(s) with the specified role per application.
```

**Review by dfunckt — COMMENTED** [review]
Thanks for the quick turnaround! Overall looks good, but as you said, let's wait for the feature to be released before merging/deploying this.

**Review by vipulgupta2048 — COMMENTED** [review]
We need to enforce verbs for whenever roles and access get mentioned.

Hence, for roles --> we use assign (assigning roles, assign roles, assigned roles)
for access --> we use grant (Granting access, granted access, grant access)_
User org will be referenced as `default organization`.

@dfunckt @pdcastro if you folks can check the most recent commit's diff. I have made the same changes + some more modifications to improve clarity. To remove the "getting started guide" feel from the document, we can weed out any first or second-person references and replace them with 3rd person nouns or references. Which can be done later.

**Comment by vipulgupta2048** [comment]
@balena-ci rebase

**Comment by dfunckt** [comment]
> User org will be referenced as default organization.

There is literally zero need for singling out that org -- there is nothing special about it and everything that applies to every org applies to that as well, therefore anything that could be said about it can be generalised and discussed in the context of orgs in general.

Agreed on your other points.

**Comment by vipulgupta2048** [comment]
@dfunckt if that's the case then I can remove that reference and I agree with your point of it being no different than anything else. It's just that on my first try, the concept and reason of having a user being converted to an organization was confusing. Hence, I felt the urge to put it forward in a better way.  That's also the reason I wrote that little section to make folks understand the idea behind it. But on second thought maybe the blog should cover the **Why** aspect and we should focus purely on being a reference on how to get things done.

**Comment by dfunckt** [comment]
Users have not been converted to organizations. What happened is that we moved their applications and billing to an org we created on their behalf. The blurb I suggested in my review yesterday -- that we automatically create an org during user registration -- is all that is needed to be said about it.

**Comment by dfunckt** [comment]
That said, I agree the blog post could briefly mention what happened in the migration of existing users -- ie. that we creqted an org for themselves and moved their apps there and added them as admins of that org.

**Comment by vipulgupta2048** [comment]
@balena-ci rebase

**Comment by pdcastro** [comment]
@vipulgupta2048, is the PR ready to be merged? (Are you waiting for anyone else's feedback?)

**Comment by vipulgupta2048** [comment]
@pdcastro It's good to go from my side.

**Review by pdcastro — APPROVED** [review]
